### PR TITLE
Remove psammead-amp-geo

### PR DESCRIPTION
**Pull request by HarryVerhoef**
**Overall change:** Removes psammead-amp-geo from psammead now that it has been migrated to Simorgh and is pending NPM deprecation.

**Code changes:**

- Removes the psammead-amp-geo code.
- Removes psammead-amp-geo from package.json.
- Remove psammead-amp-geo from the list of packages.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-amp-geo/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
...
 
+# Deprecation notice - this component is no longer being maintained.
```
Comment (andrewscfc) [inline packages/components/psammead-amp-geo/README.md]: Do we need to use this utility: https://docs.npmjs.com/cli/v7/commands/npm-deprecate ? I don't if we've done this for other packages we've deprecated?

How have we deprecated other packages? I think we'd be better actually deleting the code as we'll end up maintaining unused code in the future if we make a change like move to `yarn` for example.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.1.6 | [PR#4449](https://github.com/bbc/psammead/pull/4449) Add deprecation notice to psammead-amp-geo |
```
Comment (jroebu14) [inline CHANGELOG.md]: I just merged in some changes so that the root CHANGELOG is not required

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "4.1.5",
+  "version": "4.1.6",
```
Comment (jroebu14) [inline package.json]: I just merged in some changes so that the root package version bumping is not required

File: yarn.lock [hunk 4]
```diff
@@ -2,6 +2,7 @@
 # yarn lockfile v1
 
 
+
```
Comment (jroebu14) [inline yarn.lock]: if you run yarn again you can get rid of this whitespace 🙂

File: packages/components/psammead-amp-geo/README.md (hunk 1 above)
Comment (HarryVerhoef) [reply packages/components/psammead-amp-geo/README.md]: Thanks @andrewscfc!
This PR mimics this other one: https://github.com/bbc/psammead/pull/4425, couldn't find any documentation for deprecating a psammead package. Happy to use the npm utility and/or delete the code but the npm utility would need to be run by the owner of the package.

File: CHANGELOG.md (hunk 2 above)
Comment (HarryVerhoef) [reply CHANGELOG.md]: Awesome, thanks @jroebu14!

File: packages/components/psammead-amp-geo/README.md (hunk 1 above)
Comment (andrewscfc) [reply packages/components/psammead-amp-geo/README.md]: I'll broaden this conversation a bit, I'm a bit concerned about us accidently maintaining dead code

File: yarn.lock [hunk 5]
```diff
@@ -9249,7 +9249,7 @@ jest-worker@^26.2.1, jest-worker@^26.6.2:
     merge-stream "^2.0.0"
     supports-color "^7.0.0"
 
-jest@^26.5.0, jest@^26.6.3:
+jest@^26.5.0:
```
Comment (HarveyPeachey) [inline yarn.lock]: Hmm, is this an intended removal of `jest@^26.6.3`?

Comment (HarryVerhoef) [reply yarn.lock]: Good spot, not intentional but this change occurs whenever `yarn` is run, and does so on latest for me too. Will remove this change for now 👍

**Review by HarveyPeachey — APPROVED** [review]
Dank 👍

**Review by andrewscfc — APPROVED** [review]

**Review by simonsinclair — CHANGES_REQUESTED** [review]
The diffs on my end are showing files as deleted. For past package deprecations, we started by adding a notice to the package readme to allow a short grace period. Are we skipping this phase?

**Comment by HarryVerhoef** [comment]
> The diffs on my end are showing files as deleted. For past package deprecations, we started by adding a notice to the package readme to allow a short grace period. Are we skipping this phase?

Thanks @simonsinclair

This PR originally did exactly that, but following a discussion with @andrewscfc and @jroebu14, we thought it might be better to first get an NPM admin to deprecate the package and then delete the code in order to prevent accidentally releasing new versions of deprecated components. Up for discussion though 👍

**Review by simonsinclair — APPROVED** [review]
> This PR originally did exactly that, but following a discussion with @andrewscfc and @jroebu14, we thought it might be better to first get an NPM admin to deprecate the package and then delete the code in order to prevent accidentally releasing new versions of deprecated components. Up for discussion though 👍

Ah! Thanks for clarifying, Harry.

**Pull request merged**